src/Domain/Entity/Genre.php

```diff
@@ -66,6 +71,11 @@ public function getAncestry(): array
         return $this->ancestry;
     }
 
+    public function getChildren(): ?array
```

Comment:
This needs to throw an DataNotFetchedException if $this->children is null rather than return null